Framework/Algorithms/src/RebinByTimeAtSample.cpp

```diff
@@ -76,11 +78,11 @@ void RebinByTimeAtSample::doHistogramming(IEventWorkspace_sptr inWS,
                                      tofOffset);
```

Comment:
I did but I was worried the extra overhead in allocation and copying might result in a performance hit. ???